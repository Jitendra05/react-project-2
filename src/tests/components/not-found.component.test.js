import React from 'react';
import NotFoundComponent from '../../components/not-found.component';
import {shallow} from 'enzyme';

test('should render NotFoundComponent component correctly',()=>{
    const wrapper = shallow(<NotFoundComponent />);
    expect(wrapper).toMatchSnapshot();
});
