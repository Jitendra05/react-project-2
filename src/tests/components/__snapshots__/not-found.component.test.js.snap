// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundComponent component correctly 1`] = `
<div
  className="container"
>
  <h3>
    404:
  </h3>
   
  <Link
    to="/"
  >
    Go To Expanse Dashboard Page
  </Link>
</div>
`;
